services/order.py

```diff
@@ -0,0 +1,35 @@
+import datetime
+from typing import Optional
+
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, User, Ticket
+
+
+@transaction.atomic()
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: Optional[datetime.datetime] = None
+) -> None:
+    user = User.objects.get(username=username)
+    order = Order.objects.create(user=user)
+
+    if date:
+        order.created_at = date
+        order.save()
+
+    for ticket in tickets:
+        Ticket.objects.create(
+            order=order,
+            movie_session_id=ticket["movie_session"],
+            row=ticket["row"],
+            seat=ticket["seat"]
+        )
+
+
+def get_orders(username: Optional[str] = None) -> Order | QuerySet[Order]:

```

Comment:
Typehint of return values is incorrect. Order.objects.filter returns QuerySet too.